content/pages/exec.md

```diff
@@ -7,40 +7,29 @@ These are the people making things happen!
 
 You can view [past terms' committees here]({filename}/pages/past-exec.md).
 
-# Fall 2015 Committee #
+# Winter 2016 Committee #
 
 ## Chair ##
 
-Andee Liao (`xqliao`)
+Fatema Boxwala (`fhboxwal`)
 
-## Big CSters Director ##
-
-Jessica Yuwono (`jryuwono`)
-
-## Events Coordinator ##
-
-Ziwei Zhao (`z86zhao`)
-
-## Director of Finance ##
```

Comment:
Finance director is missing. Indeed, I think we are missing 3 committee members, as only 7 are listed here.